hivemind_audio_binary_protocol/protocol.py

```
@@ -94,7 +63,7 @@ def end_listen_callback(cls):
         LOG.info("New loop state: WAITING_WAKEWORD")
         cls.bus.emit(Message("recognizer_loop:record_end"))
 
-    def error_callback(cls, audio: sr.AudioData):
+    def error_callback(cls, audio: AudioData):
```

Ruff (0.14.13)

Unused method argument: `audio`

(ARG002)